website_docs/getting_started/browser.md

```diff
@@ -38,7 +45,7 @@ Copy the following file into an empty directory and call it `index.html`.
 </html>
 ```
 
-## Installation
+### Installation
 
 To create traces in the browser, you will need `@opentelemetry/web`, and the plugin `@opentelemetry/plugin-document-load`:
```

Comment:
Shouldn't this be `@opentelemetry/instrumentation-document-load` also should we replace mentions of plugin to instrumentation?

Reply:
Good catch!